test/e2e/specs/default-config/recordedit/add-x-forms.spec.js

```diff
@@ -213,6 +213,7 @@ describe('Record Add', function() {
                     return chaisePage.recordEditPage.getSelectAllPopupBtn(colName).click();
                 }).then(function () {
                     // wait for modal rows to load
+                    browser.wait(EC.visibilityOf(chaisePage.recordEditPage.getRecordSetTable()), browser.params.defaultTimeout);
```

Comment:
In your other PR you did this by adding a different wait condition
```js
browser.wait(EC.visibilityOf(chaisePage.recordEditPage.getCreateNewButton()), browser.params.defaultTimeout);
```
Which is more accurate? (there's a git conflict because of this change)